core/src/test/java/org/elasticsearch/action/search/SearchResponseTests.java

```diff
@@ -78,31 +79,71 @@ protected NamedXContentRegistry xContentRegistry() {
     }
 
     private SearchResponse createTestItem(ShardSearchFailure... shardSearchFailures) {
-        SearchHits hits = SearchHitsTests.createTestItem();
+        return createTestItem(false, shardSearchFailures);
+    }
+
+    /**
+     * This SearchResponse doesn't include SearchHits, Aggregations, Suggestions, ShardSearchFailures, SearchProfileShardResults
+     * to make it possible to only test properties of the SearchResponse itself
+     */
```

Comment:
@javanna @tlrx I'd be in interested in your opinion on this. I think it makes sense to take out some complexity of the test here to defer the more detailed tests for those tests to the respective other unit test.